doc/ref/gappkg.xml

```diff
@@ -997,6 +997,57 @@ is now a needed package for &GAP; itself.
 </Section>
 
 
+<!-- %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% -->
+<Section Label="Extensions Provided by a Package">
+<Heading>Extensions Provided by a Package</Heading>
+
+Sometimes a package <C>A</C> can provide additional functionality,
+such as better methods or additional data,
+if some other packages <C>B</C>, <C>C</C>, etc. are loaded.
+However, one wants that package <C>A</C> can be used also without these
+packages,
+and therefore <C>B</C>, <C>C</C>, etc. shall not be regarded as needed
+packages (see Section <Ref Sect="Package dependencies"/>) of <C>A</C>.
+
+<P/>
+
+One way to deal with this situation is to put those parts of code of <C>A</C>
+that depend on <C>B</C>, <C>C</C>, etc., into files that get read only
+in the situation that the packages in question have actually been loaded
+into the current &GAP; session,
+and to notify this conditional <E>extension</E> of <C>A</C> via a record in
```

Comment:
The extension is not really "notified" of anything, is it? It get loaded.

Or maybe you meant to say that GAP is informed about the existence of such an extension? So maybe:
```suggestion
and to signal the existence of this conditional <E>extension</E> of <C>A</C> via a record in
```